frontend/src/components/entity/EntityMetadata.tsx

```diff
@@ -41,58 +54,73 @@ export const VideoMetadata = ({
     <Box
       sx={{
         display: 'flex',
-        flexWrap: 'wrap',
+        flexWrap: flexWrap,
         alignContent: 'space-between',
         fontSize: '0.8rem',
         color: 'neutral.main',
-        columnGap: '12px',
+        columnGap: '10px',
```

Comment:
Is it still relevant? Now that the number of views is abbreviated I think I would prefer to preserve the same gap (also to avoid confusion with the date that comes just before)

```suggestion
        columnGap: '12px',
```